plugin/evm/customheader/block_gas_cost.go

```diff
@@ -117,3 +120,70 @@ func EstimateRequiredTip(
 	estimatedTip := totalRequiredTips.Div(totalRequiredTips, totalGasUsed)
 	return estimatedTip, nil
 }
+
+func VerifyBlockFee(
+	baseFee *big.Int,
+	requiredBlockGasCost *big.Int,
+	txs []*types.Transaction,
+	receipts []*types.Receipt,
+	extraStateChangeContribution *big.Int,
```

Comment:
Should we remove `extraStateChangeContribution` as well? We don't currently have it in subnet-evm.

I'm fine leaving it if its to reduce the diff with coreth, but I want to be intentional about it.